txcache/selectionTracker_test.go

```diff
@@ -202,3 +225,67 @@ func TestSelectionTracker_removeFromTrackedBlocks(t *testing.T) {
 
 	require.Equal(t, expectedTrackedBlock, selTracker.blocks[0])
 }
+
+func TestSelectionTracker_getTransactionsFromBlock(t *testing.T) {
```

Comment:
Test is good for now, but, as mentioned, I think we'll rework that function (and get the transactions from the storage).